mqtt/build/linux/buildImage.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# This script copies the iodedged executable files that go into the azureiotedge-iotedged image,
+# for each supported arch (x86_64, arm32v7, arm64v8).
+# It then publishes executable files along with their corresponding dockerfiles to the publish directory,
+# so that buildImage.sh can build the container image.
+
+set -e
+###############################################################################
+# Define Environment Variables
+###############################################################################
+
+ARCH=$1
```

Comment:
I think we may want to have the same structure for parsing script arguments and have them with named keys